ocaml/xapi-cli-server/records.ml

```diff
@@ -2431,6 +2431,9 @@ let vm_record rpc session_id vm =
               (x ()).API.vM_pending_guidances
           )
           ()
+      ; make_field ~name:"vtpm"
```

Comment:
The name is singular, but we're forced to print a list because I couldn't find a way to use an `option` in the datamodel.